packages/peregrine/lib/talons/CartPage/PriceAdjustments/useCouponCode.js

```diff
@@ -87,26 +91,28 @@ export const useCouponCode = props => {
         setIsCartUpdating
     ]);
 
-    let applyErrorMessage;
+    let derivedErrorMessage;
+
+    if (applyError || removeCouponError) {
+        const errorTarget = applyError || removeCouponError;
 
-    if (applyError) {
-        if (applyError.graphQLErrors) {
+        if (errorTarget.graphQLErrors) {
             // Apollo prepends "GraphQL Error:" onto the message,
             // which we don't want to show to an end user.
             // Build up the error message manually without the prepended text.
-            applyErrorMessage = applyError.graphQLErrors
+            derivedErrorMessage = errorTarget.graphQLErrors
                 .map(({ message }) => message)
                 .join(', ');
         } else {
             // A non-GraphQL error occurred.
-            applyErrorMessage - applyError.message;
+            derivedErrorMessage = errorTarget.message;
```

Comment:
Do we need to handle network errors? They'd be on the target as `errorTarget.networkError`.

Reply:
Discussed offline - I am for moving this sort of logic into a utility that is used by `useFormError` and by any other components that need to turn apollo errors into a string.

Reply:
I was going to suggest that too, 💯  agree